File: app/src/main/java/com/cookplan/recipe/view_item/RecipeView.java
package com.cookplan.recipe.view_item;

import com.cookplan.models.Ingredient;

import java.util.List;

/**
 * Created by DariaEfimova on 23.03.17.
 */

public interface RecipeView {
    void setErrorToast(String error);

    void setIngredientList(List<Ingredient> ingredientList);

    void setIngredientSuccessfulUpdate(Ingredient ingredient);

}

File: app/src/main/java/com/cookplan/recipe/view_item/RecipeView.kt
package com.cookplan.recipe.view_item

import com.cookplan.models.Ingredient

/**
 * Created by DariaEfimova on 23.03.17.
 */

interface RecipeView {

    fun setErrorToast(error: String)

    fun setIngredientList(ingredientList: List<Ingredient>)

    fun setIngredientSuccessfulUpdate(ingredient: Ingredient)

    fun ingredListChangedShoplistStatus(isRemoved: Boolean)
}
